basilisk/tests/test_accessible_output.py

```
@@ -0,0 +1,539 @@
+import pytest
```

Ruff (0.11.9)

Missing docstring in public module

(D100)